lib/ZumoOta/src/BootloaderCom.cpp

```diff
@@ -477,8 +641,45 @@ bool BootloaderCom::compareExpectedAndReceivedResponse(const uint8_t command[],
     return true;
 }
 
+ void BootloaderCom:: updateCommand(uint8_t newCommand[])

```

Comment:
generic function name that is only applicable for set_addr. 
And it doesn't belong here, but in the provider for programming,

Reply:
It will be adjusted in the next commit.